pkgs/development/python-modules/mkl-service/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchFromGitHub, cython, mkl, nose, six }:
+
+buildPythonPackage rec {
+  pname = "mkl-service";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "IntelPython";
+    repo = "mkl-service";
+    rev = "v${version}";
+    sha256 = "1bnpgx629rxqf0yhn0jn68ypj3dqv6njc3981j1g8j8rsm5lycrn";
+  };
+
+  MKLROOT = mkl;
```

Comment:
This is needed so that `setup.py` picks up the location of the `mkl` package:
https://github.com/IntelPython/mkl-service/blob/master/setup.py#L67